src/@carbon-info/views/Features/Features.tsx

```diff
@@ -0,0 +1,246 @@
+import connective from "@carbon-info/assets/animated/connective.json";
+import stable from "@carbon-info/assets/animated/stable.json";
+import versatile from "@carbon-info/assets/animated/versatile.json";
+import carbonFeaturesBackground from "@carbon-info/assets/background/carbonFeaturesBackground.svg";
+import { CTAButton, FadeAndSlide } from "@carbon-info/components";
+import { Path } from "@carbon-info/constants";
+import { Box, Theme, Typography, makeStyles, useMediaQuery, useTheme } from "@material-ui/core";
+import clsx from "clsx";
+import React from "react";
+import { useInView } from "react-intersection-observer";
+import Lottie from "react-lottie";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
+
+interface FeatureItem {
+  header: string,
+  subheader: string,
+  description: string,
+  ctaLink: string,
+  icon: any,
+}
+
+const Features: React.FC = () => {
+  const classes = useStyles();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+
+  const { ref, inView } = useInView({
+    /* Optional options */
+    threshold: 0.4,
+    triggerOnce: true,
+  });
+
+  const ConnectiveAnimation = {
+    loop: true,
+    autoplay: true,
+    animationData: connective,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const StableAnimation = {
+    loop: true,
+    autoplay: true,
+    animationData: stable,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const VersatileAnimation = {
+    loop: true,
+    autoplay: true,
+    animationData: versatile,
+    rendererSettings: {
+      preserveAspectRatio: "xMidYMid slice",
+    },
+  };
+
+  const items: FeatureItem[] = [{
+    header: "Connective",
+    subheader: "Powers interoperability",
+    description: "between multiple blockchains with on-chain liquidity through PolyNetwork and IBC.",
+    ctaLink: Path.Footer.Guides,
+    icon: ConnectiveAnimation,
+  }, {
+    header: "Versatile",
+    subheader: "Native support",
+    description: "of any crypto derivatives, amplified liquidity pools, AMMs, on-chain order books and more.",
+    ctaLink: Path.Footer.Guides,
+    icon: VersatileAnimation,
+  }, {
+    header: "Secure & Stable",
+    subheader: "Custom-built",
+    description: "using Cosmos-SDK and secured through Tendermint PoS for trustless and safe transactions.",
+    ctaLink: Path.Footer.Guides,
+    icon: StableAnimation,
+  }];
+
+  const responsive = {
+    desktop: {
+      breakpoint: { max: 3000, min: 1024 },
+      items: 1,
+    },
+    tablet: {
+      breakpoint: { max: 1024, min: 464 },
+      items: 1,
+    },
+    mobile: {
+      breakpoint: { max: 464, min: 0 },
+      items: 1,
+    },
+  };
```

Comment:
Saw a few instances of this variable. If all are constant, then would be good to store this in a constants file and import everytime you need to use.

If not, can ignore this message haha